cmd/util.go

```diff
@@ -360,3 +360,13 @@ func prettifyReason(reason string) string {
 	}
 	return reason
 }
+
+func isCSCWellDefined(podSpec PodSpecV1, container ContainerV1) bool {
+	if container.SecurityContext != nil && container.SecurityContext.RunAsNonRoot != nil {
```

Comment:
Does it make sense to check for `RunAsNonRoot != nil` because we already check it later?

Reply:
renamed the function to a better name of what it does, should be more clear now